lldb/tools/lldb-dap/Transport.cpp

```diff
@@ -0,0 +1,110 @@
+//===-- Transport.cpp -------------------------------------------*- C++ -*-===//
```

Comment:
The `*- C++ -*` was only meant to go into headers (to tell emacs they were C++ rather than C headers) but there was an RFC to drop it altogether. New files should omit it. 
```suggestion
//===-- Transport.cpp ------------------------------------------------------===//
```